### DISCUSSION TITLE
Dedicated Relational Tags Interface

### DISCUSSION THREAD

**Discussion by egeozcan** [directus/directus#6394] [Feature Requests]
Hi!

While it is perfectly possible to add a tags entity and use a many-to-many relationship, I think adding tags is such a common thing that there could be a dedicated field for it, that creates relevant tables in the background when used (tags, entity_tags etc.)

The UI can also treat tags specially in that the list and card views can get "click to filter" tags clouds.

**Comment by benhaynes** [comment]
I agree! We have a tags Interface, but could add a relational tags Interface too.

Duplicate: https://github.com/directus/directus/discussions/3065
